llvm/lib/Target/X86/X86InstrSSE.td

```diff
@@ -6698,6 +6698,30 @@ let Constraints = "$src1 = $dst" in {
     def CRC32r64r8 : SS42I_crc32r<0xF0, "crc32{b}", GR64, GR8,
                                    null_frag>, REX_W;
   }
+
```

Comment:
@KanRobert said we need `In64BitMode` even for assembler?

Reply:
Yes.

Reply:
Will add.